mobile/README.md

```diff
@@ -17,30 +17,47 @@ To add a new translation text, enter the key-value pair in the `i18n/en.json` in
 make translation
 ```
 
+## Isar models
```

Comment:
The changes in this file aren't really related, I suppose. It's just missing/deprecated documentation that I tried to update a bit. I can revert the change, if you insist.

Reply:
Please help revert the changes here

Reply:
done